### PR TITLE
feat: add dash realtimeMode

### PR DESCRIPTION
**Pull request by imechoim**
Add so-called `realtimeMode` auto-update dash scenario.
`realtimeMode` is useful in cases where dash items are updated at different rates and you cannot guarantee data consistency at a given time. `realtimeMode` is based on `Promise.allSettled`.

### CODE REVIEW

**Comment by Marginy605** [comment]
Let's check with DL logic for loading widgets in viewport only

File: src/components/GridLayout/GridLayout.js [hunk 1]
```diff
@@ -70,27 +69,50 @@ export default class GridLayout extends React.PureComponent {
...
+            });
+            return;
+        }
+
+        const autoupdateIntervalMs = this.getAutoupdateIntervalMs();
         if (autoupdateIntervalMs) {
-            const timeSinceLastReload = new Date().getTime() - (this._lastReloadAt || 0);
-            const reloadIntervalRemains = autoupdateIntervalMs - timeSinceLastReload;
-
-            if (!isPageHidden && !editMode && reloadIntervalRemains <= 0) {
```
Comment (Marginy605) [inline src/components/GridLayout/GridLayout.js:85]: Here was a logic for autoupdate for reloading immediately if dash was not in focus for time more that timer setting and later become in focus. I can't see this in PR new changes

**Review by Marginy605 — CHANGES_REQUESTED** [review]

File: src/components/GridLayout/GridLayout.js (hunk 1 above)
Comment (imechoim) [reply src/components/GridLayout/GridLayout.js:85]: Thank you for the highlight. It's not an obvious feature.
For the implementation of this feature, I used the `visibilitychange` event.

Comment (Marginy605) [reply src/components/GridLayout/GridLayout.js:85]: Now there is bug that if we return earlier that interval ended we update it, but we need to wait for rest of time

**Pull request closed without merging**